lib/trento/application/projectors/sap_system_projector.ex

```diff
@@ -207,11 +209,14 @@ defmodule Trento.SapSystemProjector do
         _,
         _
       ) do
+    %SapSystemReadModel{sid: sid} = Repo.get!(SapSystemReadModel, sap_system_id)
```

Comment:
Doesn't this raise some any failure in tests?
We should have a look, or at least try to include this change in the tests. 
Maybe the assertion looks for partial objects